src/styles/card.css

```diff
@@ -0,0 +1,42 @@
+* {
+  box-sizing: border-box;
+}
+
+.card {
+  display: flexbox;
+  flex-wrap: wrap;
+  justify-content: center;
+  padding-right: 16px;
+  padding-left: 16px;
+  font-family: "Roboto", sans-serif;
+  font-size: 16px;
+  max-width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  margin-top: 32px;
+  margin-left: 3px;
```

Comment:
![image](https://user-images.githubusercontent.com/96409563/197149152-da5d0d4a-6ed2-442f-a059-1274d97628ab.png)
Please fix all cases to avoid possible [margin collapse](https://www.w3schools.com/css/css_margin_collapse.asp)